static/index.html

```diff
@@ -316,11 +337,39 @@ <h2 class="response-title">Test Ether sent to the wallet</h2>
       function handle_response(address) {
         return function (data) {
           if (!data.error) {
+            const tx_id  = `0x${data.tx_id}`;
+            const link = document.getElementById(
+              "explorer-link"
+            );
             document.getElementById("form").hidden = true;
             document.getElementById("response").style.display = "block";
-            document.getElementById(
-              "explorer-link"
-            ).href = `${blockExplorer}/tx/0x${data.tx_id}`;
+            link.href = `${blockExplorer}/tx/${tx_id}`;
+
+            if (shouldAutoClose) {
+              window.close();
+            }
+            
+            if (redirectUrl) {
+              try {
+                const refUrl = new URL(decodeURIComponent(redirectUrl));
+                const redirectText = `Redirecting you back to ${refUrl.origin}...`;
+                
+                // Add tx_id to query params
+                refUrl.searchParams.set('tx_id', tx_id);
+
+                link.classList.add('hidden');
+                const $redirectContainer = document.getElementById('redirect-container');
+                const $redirectText = document.getElementById('redirect-text');
+                $redirectContainer.classList.remove('hidden');
+                $redirectText.innerText = redirectText;
+                setTimeout(() => {
+                  location.href = refUrl.toString();
```

Comment:
Is this how you change the current page you're on?

Reply:
yes, once you assign a new value for a href or domain, it redirects the user.